packages/tests/src/utils/nodes.ts

```diff
@@ -57,9 +66,10 @@ export async function runMultipleNodes(
   for (const node of serviceNodes.nodes) {
     await waku.dial(await node.getMultiaddrWithId());
     await waku.waitForPeers([Protocols.Filter, Protocols.LightPush]);
-    await node.ensureSubscriptions(
-      derivePubsubTopicsFromNetworkConfig(networkConfig)
-    );
+    // TODO
```

Comment:
did this `ensure` failed?

Reply:
no, it's a clean up needed (one does not derive pubsub topics from a network config anymore)